src/db/models/legacy.ts

```diff
@@ -0,0 +1,15 @@
+import * as t from 'io-ts';
+
+import { defineSequelizeModel } from '../util/sequelize-model';
+
+export default defineSequelizeModel({
+  tableName: 'legacy',
+  fields: {
+    required: {
+      objectType: { kind: 'checked', type: t.string },
+      objectID: { kind: 'checked', type: t.number },
+      legacyID: { kind: 'checked', type: t.number },
+    },
+  },
+  softDeletionEnabled: false,
+});
```

Comment:
Model for this table wasn't defined on purpose. As its name suggests, this table is storing legacy data, which we no longer want. I see that newest record of this table is from January 2019, which is more than 5 years old.

On the other hand, I see that this model is used in https://github.com/UN-OCHA/hpc-api/pull/219, but I am not sure whether we truly need it or not. Maybe we need to use it for the sake of completeness of the API, to support old flows. If you are sure we need to use it, then keep this model, but change the type of `objectType` to `FLOW_OBJECT_TYPE_TYPE`.
```suggestion
export default defineSequelizeModel({
  tableName: 'legacy',
  fields: {
    required: {
      objectType: { kind: 'checked', type: FLOW_OBJECT_TYPE_TYPE },
      objectID: { kind: 'checked', type: t.number },
      legacyID: { kind: 'checked', type: t.number },
    },
  },
  softDeletionEnabled: false,
});
```

Reply:
As peer as your comment this is needed to provide full compatibility with the previous version of fts-admin.
fts-admin v2 must provide pair-functionality with fts-admin.